charts/firefly/values.yaml

```diff
@@ -261,7 +266,8 @@ core:
     #    hosts:
     #      - chart-example.local
 
-  resources: {}
+  resources:
+    {}
     # We usually recommend not to specify default resources and to leave this as a conscious
```

Comment:
what are all these indents? some sort of linting?

Reply:
Yep, it's some linting that's causing this but I can't recreate that linting on my own machine either through the `make lint` commands or by running those commands directly against the directory so I'm unsure how this was in the original PR. Both are obviously valid, but the original version is significantly more conventional so going to back out these changes.

Reply:
Changed in https://github.com/hyperledger/firefly-helm-charts/pull/76/commits/36bd1b29bf4ab472d68a82d5075b5a3f247ed74b